Run/phpunit.xml.dist

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit colors="true">
+  <testsuites>
+    <testsuite>
```

Comment:
I've been seeing a warning for `testsuite` blocks without the name attributes, I was thinking if we should start adding them. But obviously not critically important.